components/ShareButton.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { Share, View, Button } from 'react-native';
+
+// Styling
+import {
+  ActiveRoundedButton,
+  ActiveButtonText,
+} from '../styling-components/Global';
+
+// ----------------------------------------------------------------
+
+const ShareExample = ({ navigation, SharableMessage }) => {
+  const onShare = async () => {
```

Comment:
Great job with async await :)
For further explanation of this topic, check out https://medium.com/javascript-in-plain-english/async-await-javascript-5038668ec6eb